src/p5xr/core/p5xrButton.js

```diff
@@ -140,7 +140,7 @@ export default class p5xrButton {
    */
   setDevice(device) {
     this.device = device;
-    this.__updateButtonState();
+    // this.__updateButtonState();
```

Comment:
Can be removed. See other comments.

Reply:
Removed 033099c5f85e42f452279ff69fa6e2c8992086ca